### PR TITLE
feat(register) : adding register pages

### PR DESCRIPTION
**Pull request by SpiritGY**
# Description

Register pages for medecin app

### Click Up reference

[CU-86c0hjvby](https://[https://app.clickup.com/t/86c0hjvby)

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Adding 2 register pages build like Figmas
adding register services
deleting Maj on first letter on password field

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to a clickup task
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

**Review by lucas-louis — CHANGES_REQUESTED** [review]
- Too much padding at the bottom of the onboarding page
- Can't create a doctor => error "Les identifiants ne correspondent pas" (maybe because the email and password input are not stored?)

File: medecin_mobile/lib/services/login_service.dart [hunk 1]
```diff
@@ -171,6 +172,9 @@ Future<void> register(Map<String , dynamic> dInfo, BuildContext context) async {
     }),
   );
 
+  Logger().d(response.statusCode);
+  Logger().d(response.body);
```
Comment (lucas-louis) [inline medecin_mobile/lib/services/login_service.dart:176]: 🤨

**Review by lucas-louis — CHANGES_REQUESTED** [review]

**Review by lucas-louis — APPROVED** [review]
LGTM

**Review by MarvinFlm — APPROVED** [review]
LGTM

**Pull request merged**